x-pack/plugins/fleet/common/types/rest_spec/agent.ts

```diff
@@ -220,6 +220,7 @@ export interface GetAgentStatusResponse {
 export interface GetAgentIncomingDataRequest {
   query: {
     agentsIds: string[];
+    pkgKey?: string;
```

Comment:
this was updated to `pkgName` and `pkgVersion` - I think you commented on an outdated version. it is used by this API to check if agent(s) have ingested data for only that package's data streams (instead of *any* data ingested by that agent)